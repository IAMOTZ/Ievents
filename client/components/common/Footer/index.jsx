import React from 'react';

const footerStyle = {
  backgroundColor: 'rgba(0,0,0, 0.9)',
  borderTop: '2px solid rgb(0, 142, 214)'
};

const Footer = () => (
  <footer className="w-100 mt-auto" style={footerStyle}>
    <div className="container text-white text-center py-5">
      <h1>iEvents</h1>
      <p>Copyright &copy; 2017</p>
    </div>
  </footer>
);

export default Footer;
