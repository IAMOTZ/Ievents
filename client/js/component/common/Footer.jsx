import React from 'react';

const Footer = () => (
  <footer className="w-100 mt-auto">
    <div className="container text-white text-center py-5">
      <h1>Ievents</h1>
      <p>Copyright &copy; 2017</p>
    </div>
  </footer>
);

export default Footer;
