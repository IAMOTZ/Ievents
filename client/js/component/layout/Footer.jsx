import React from 'react';

export default () => {
  return (
    <footer className="w-100 mt-auto">
      <div className="container text-white text-center py-5">
        <h1>Ievents</h1>
        <p>Copyright &copy; 2017</p>
      </div>
    </footer>
  );
};
