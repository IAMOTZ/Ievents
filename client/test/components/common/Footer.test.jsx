import React from 'react';
import Footer from '../../../components/common/Footer';

describe('<Footer />', () => {
  describe('Rendering:', () => {
    it('should render correctly', () => {
      const wrapper = shallow(<Footer />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
