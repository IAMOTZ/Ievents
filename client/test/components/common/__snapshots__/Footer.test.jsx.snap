// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Rendering: should render correctly 1`] = `
<footer
  className="w-100 mt-auto"
  style={
    Object {
      "backgroundColor": "rgba(0,0,0, 0.9)",
      "borderTop": "2px solid rgb(0, 142, 214)",
    }
  }
>
  <div
    className="container text-white text-center py-5"
  >
    <h1>
      iEvents
    </h1>
    <p>
      Copyright Â© 2017
    </p>
  </div>
</footer>
`;
